spec/acceptance/redis_cli_task_spec.rb

```diff
@@ -4,7 +4,7 @@
 
 describe 'redis-cli task' do
   subject do
-    on(default, "bolt task run --modulepath /etc/puppetlabs/code/modules --targets localhost #{task_name} #{params}", acceptable_exit_codes: [0, 1]).stdout
+    on(default, "bolt task run --modulepath /etc/puppetlabs/code/environments/production/modules --targets localhost #{task_name} #{params}", acceptable_exit_codes: [0, 1]).stdout
```

Comment:
Is it still needed to specify the modulepath at all? Isn't this the default built in?

Reply:
Default was `/root/.puppetlabs` unfortunately.

Reply:
Perhaps works, and let Puppet interpolate it. Not sure how much escaping is needed.
```suggestion
    on(default, "bolt task run --modulepath '/etc/puppetlabs/code/environments/$envirornment/modules' --targets localhost #{task_name} #{params}", acceptable_exit_codes: [0, 1]).stdout
```

Reply:
Tried various level of escaping and brackets with `$environment`

Tried `%x(puppet config print modulepath)` in various ways